### ISSUE TITLE
Supressing libcurl warnings

### ISSUE THREAD

**Issue by wesm** [conda-forge/openssl-feedstock#14]
In my Travis CI builds, I'm getting warnings like:

```
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libcrypto.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
[  5%] Built target arrow_jemalloc_objlib
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libcrypto.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
[  8%] /usr/bin/cmake/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libcrypto.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libcrypto.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
[ 10%] /usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libssl.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
/usr/bin/cmake: /home/travis/build/apache/arrow/parquet-env/lib/libcrypto.so.1.0.0: no version information available (required by /usr/lib/x86_64-linux-gnu/libcurl.so.4)
```

It suggests I should be using a cmake and curl/libcurl all from conda-forge so that the openssl version matches. What is the best practice for libraries to conda install to avoid this?

**Comment by jakirkham** [comment]
> It suggests I should be using a cmake and curl/libcurl all from conda-forge so that the openssl version matches.

Yep, sounds reasonable. Would run the command below during your setup. This will stick `conda-forge` as highest priority. Additional runs of `conda config --add channels` afterwards will promote channels above `conda-forge`. So ordering any other calls to `conda config --add channels` is important.

```
$ conda config --add channels conda-forge
```

**Comment by jakirkham** [comment]
Is this still an issue, @wesm?

**Comment by wesm** [comment]
I've been solving this by installing curl, git, and cmake all from conda-forge and placing them first in PATH/LD_LIBRARY_PATH

**Comment by carlodri** [comment]
seems like this can be closed

**Comment by matheusdiogenesandrade** [comment]
If it helps, I posted a solution for this problem in this [question](https://askubuntu.com/questions/830466/libcrypto-so-1-0-0-no-version-information-available-required-by-ssh/1198404#1198404).
